src/components/button/index.md.njk

```diff
@@ -46,8 +46,8 @@ Avoid using multiple default buttons on a single page. Having more than one main
 
 ### Start buttons
 
-Use a start button for the main call to action on your service’s [start page](/patterns/start-pages/). 
-Start buttons use the <a> element rather than the <button> element, because they don't submit any data. 
+Use a start button for the main call to action on your service’s [start page](/patterns/start-pages/).
+Start buttons use the `<a>` element rather than the `<button>` element, because they don't submit any data.
 They are given button styling and behaviour because they are the main call to action on the page.
```

Comment:
small thing but this is passive voice - active might be:

"We use button styling and behaviour because Start buttons are the main call to action on the page."